yjit_codegen.c

```diff
@@ -3440,25 +3440,6 @@ gen_return_branch(codeblock_t *cb, uint8_t *target0, uint8_t *target1, uint8_t s
     }
 }
 
-// Returns whether the iseq only needs positional (lead) argument setup.
-static bool
-iseq_lead_only_arg_setup_p(const rb_iseq_t *iseq)
-{
-    // When iseq->body->local_iseq == iseq, setup_parameters_complex()
-    // doesn't do anything to setup the block parameter.
```

Comment:
Removal of this comment without replacement was a red flag for me and I pushed 4efe979d4b595d6599dc91b4d9a3b960aa87c47c which I think is better. Please take a look and I hope you don't mind me pushing to your branch unprompted.

Reply:
Looks great!